graspologic/nominate/VNviaSGM.py

```diff
@@ -0,0 +1,387 @@
+import numpy as np
+from ..match import GraphMatch as GMP
+from sklearn.base import BaseEstimator
+import itertools
+import warnings
+
+
+class VNviaSGM(BaseEstimator):
+    """
+    This class implements Vertex Nomination via Seeded Graph Matching
+    (VNviaSGM) with the algorithm described in [1].
+
+    VNviaSGM is a nomination algorithm, so instead of completely matching two
+    graphs `A` and `B`, it proposes a nomination list of potential matches in
+    graph `B` to a vertex of interest (voi) in graph `A` with associated
+    probabilities. VNviaSGM generates an initial induced subgraph about the
+    voi to determine which seeds are close enough to be used.
+    All the seeds that are close enough are then used to generate subgraphs in
+    both A and B. These subgraphs are matched using the seeded graph matching
+    algorithm (SGM), and a nomination list is returned.
+
+    Parameters
+    ----------
+    order_voi_subgraph: int, positive (default = 1)
+        Order used to create induced subgraph on `A` about voi where the max
+        distance between voi and other nodes is order_voi_subgraph.
```

Comment:
double backtick parameters and attributes. do this for all such references
```suggestion
        distance between voi and other nodes is ``order_voi_subgraph``.
```